scripts/remove

```diff
@@ -74,7 +74,7 @@ ynh_script_progression --message="Removing Tvheadend main directory and config f
 
 # Remove the app directory securely (/home/hts folder)
 ## Use rm -r because ynh_secure_remove says that /home/hts is not an acceptable path to delete
-rm -r $final_path
+ynh_secure_remove --file="$final_path"
 ynh_secure_remove --file="/etc/default/tvheadend"
```

Comment:
Hi Eric, thank for the PR.
But as you can see on my comment here, yah_secure_remove does not work in this case 😕

Reply:
ok thanks, I missed your comment.
Indeed, `ynh_secure_remove` helper prevents from removing `/home/*` path...
[ynh_secure_remove](https://github.com/YunoHost/yunohost/blob/71a08c09ea5099d7951d8d7badf0ed3bc0a02cf2/data/helpers.d/utils#L715)

Reply:
Maybe @alexAubin has an idea of a cleaner way to fix this linter warning?

Reply:
ping @alexAubin